iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -917,26 +981,24 @@ static void decideFusableLinalgOps(FuncOp funcOp) {
       // Start with a root operation and fuse its producers.
       Operation *op = linalgOp.getOperation();
       if (!isRootOp(op)) continue;
-      if (op->getAttrOfType<ArrayAttr>(kFusionGroupsAttr)) continue;
-      unsigned currGroupNum = numRootOps++;
-      op->setAttr(kRootOpAttr, builder.getI64IntegerAttr(currGroupNum));
-      for (auto operand : linalgOp.getShapedOperands()) {
-        auto producer = operand.getDefiningOp<linalg::LinalgOp>();
+
+      // This might be a root op that is already fused into another root op.
```

Comment:
I think it might be better to add a flag that guards when we chose to fuse elementwise operations with the named ops. There is a bit of an issue with aggressive fusion (number of descriptors used, etc. etc.). We can for now leave the flag on by default. Eventually we need to have some arbitrary heuristic to limit fusion (number of operands of dispatch region, for example), but that is not needed right here.

Reply:
Added a CL option to disable operand fusion.